### ISSUE TITLE
Geometric primitives can't be cimported

### ISSUE THREAD

**Issue by jacklovell** [raysect/source#285]
I'd like to be able to do the following:
`from raysect.primitive cimport Mesh, Cylinder, Cone, Intersect, Subtract, Union`
so that I can instantiate many of these primitives in a loop without having to go through the python layer every time. However, I can't find an `__init__.pxd` where these primitives are defined. They're all `cdef class` so I should in theory be able to `cminport` them if an `__init__.pxd` was provided. Is there a reason this hasn't already been done?

**Comment by mattngc** [comment]
This is definitely an Alex issue. Will let him comment here before I try to do anything.

**Comment by CnlPepper** [comment]
I've added the missing pxd files, they will be available in the next release.

I've also given some thought to adding the new_* methods we discussed in person. I'm not going to add them at this stage as it will require significant refactoring of the initialisation of each primitive. It would be best to defer this until the next major release as it might result in some minor changes to the initialisation routines in python. I'm not comfortable making the change in a minor release.
